WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -123,7 +124,8 @@
           <Border Classes="warning">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction uses non-private coins." />
+              <TextBlock Text="Transaction uses non-private coins."
+                         ToolTip.Tip="Some entities can deanonymize this transaction because its privacy is as good as the used coins' lowest privacy. Coinjoin more to have (enough) private coins for this transaction." />
```

Comment:
```suggestion
                         ToolTip.Tip="Some entities can deanonymize this transaction because it spends coins with low privacy. Coinjoin more to have (enough) private coins for this transaction." />
```

Reply:
This one is the non-private coins case, so those coins have `no` privacy at all, so I guess it should be: `because it spends coins with no privacy`

Reply:
done. applied your sugestions with Yahia's improvements